NAMESPACE

```diff
@@ -5,6 +5,7 @@ S3method(civis_ml,character)
 S3method(civis_ml,civis_file)
 S3method(civis_ml,civis_table)
 S3method(civis_ml,data.frame)
+S3method(coef,civis_ml)
 S3method(db_query_fields,CivisConnection)
```

Comment:
@patr1ckm Does this look right? Is this supposed to be an `S3method` ?

Reply:
Oh, `coef` is already a generic method defined in `stats`, we shouldn't re-implement here. See `stats::coef`